drivers/display/display_ili9806e_dsi.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_DISPLAY_ILITEK_ILI9806E_H_
+#define ZEPHYR_DRIVERS_DISPLAY_ILITEK_ILI9806E_H_
+
+/* Horizontal low pulse width */
+#define ILITEK_ILI9806E_HSYNC 2U
```

Comment:
Are these hardcoded? Seems like it would vary based on the panel the controller was driving, and like these should be in devicetree.

Reply:
Done. This configuration has been moved to dts